web_src/less/_repository.less

```diff
@@ -2177,8 +2177,10 @@
 
             > .item {
                 float: left;
-                padding: .5rem .5rem !important;
```

Comment:
I've opened https://github.com/go-gitea/gitea/pull/11436 for this, you can remove the change here.